src/core/actions/index.js

```diff
@@ -49,6 +50,18 @@ export function setLang(lang: string): SetLangAction {
   };
 }
 
+export type SetRegionCodeAction = {|
+  payload: {| regionCode: string |},
+  type: typeof SET_REGION_CODE,
+|};
+
+export function setRegionCode(regionCode: string): SetRegionCodeAction {
+  return {
+    type: SET_REGION_CODE,
+    payload: { regionCode },
+  };
+}
```

Comment:
I know it's not part of the feature but given that we're going to touch this file, maybe it would be time to unify the different files into a single [duck](https://github.com/erikras/ducks-modular-redux).

Reply:
Yeah, I suppose we could. Makes for a bigger diff, but it's pretty simple stuff.